contracts/test/DepositStub.sol

```diff
@@ -77,4 +78,8 @@ contract DepositStub is IDeposit {
     function withdrawableAmount() external view override returns (uint256) {
         return address(this).balance;
     }
+
```

Comment:
What do you think about separating this function from the rest for clarity what comes from tBTC and what was added by us? Something like:
```
    //
    // Not in tBTC deposit interface, functions below were added just for tests.
    //

    function setAuctionValue(uint256 _auctionValue) external {
        auctionValue = _auctionValue;
    }
```

Reply:
https://github.com/keep-network/coverage-pools/pull/79/commits/a7745dcdc76f981f1d1643f861db76b94abcc422